TestApp/android/app/build.gradle

```
@@ -169,7 +169,8 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation 'com.facebook.react:react-native:+'
-    
+    implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
```

```suggestion
    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.0.0'
```

Also what is that dependency?